homeassistant/components/xiaomi_aqara/__init__.py

```diff
@@ -247,8 +247,8 @@ def available(self):
 
     @property
     def should_poll(self):
-        """Return the polling state. No polling needed."""
-        return False
+        """Return the polling state."""
+        return True
```

Comment:
If we should return True here, we should remove the property completely, since True is the default.

Reply:
Thanks, I'd removed these lines